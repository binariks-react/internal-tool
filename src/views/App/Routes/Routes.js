import React from 'react';
import { Switch, Route } from 'react-router-dom';
import TestPage from 'views/Pages/TestPage/TestPageContainer';
import ErrorPage from 'views/Pages/ErrorPage/ErrorPageContainer';

const routes = [
  {
    path: '/',
    exact: true,
    component: TestPage,
  },
  {
    path: '/error',
    exact: true,
    component: ErrorPage,
  },
];

const Routes = () => (
  <Switch>
    {
      routes.map(r => (
        <Route
          key={r.path}
          path={r.path}
          exact={r.exact}
          component={r.component}
        />))
    }
  </Switch>
);

export default Routes;
