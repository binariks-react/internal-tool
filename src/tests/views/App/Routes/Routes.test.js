import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Routes from 'views/App/Routes/Routes';

describe('Routes', () => {
  it('snapshot', () => {
    const routes = shallow(
      <Routes />
    );
    expect(toJson(routes)).toMatchSnapshot();
  });
});
