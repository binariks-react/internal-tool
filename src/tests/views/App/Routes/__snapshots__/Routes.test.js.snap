// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes snapshot 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    key="/"
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    key="/error"
    path="/error"
  />
</Switch>
`;
